xapian-letor/docs/letor.rst

```diff
@@ -30,14 +30,15 @@ Learning the model
 As mentioned before, this process requires a training file in the above format. xapian-letor API empowers you to generate such training file. But for that you have to supply some information files like:
 
     1. Query file: This file has information of queries to be involved in
-    learning and its id. It should be formatted in such a way::
+    learning and its id. Here space is used as delimiter betweem query id and query string.
+    It should be formatted in such a way::
 
       2010001 'landslide malaysia'
       2010002 'search engine'
       2010003 'Monuments of India'
       2010004 'Indian food'
 
-    where 2010xxx being query-id followed by a comma separated query in
+    where 2010xxx being query-id followed by a space separated query in
```

Comment:
It's can be space separated or comma separated query as @ojwb mentioned. "followed by a query in single quotes"

Reply:
We will fix this and more issues in the documentation in a different PR.

Reply:
@AyushP123 Which other issues are you talking about?.

Reply:
I should have framed it better: We will fix this along with other with other issues in the documentation, which will form as we keep on developing letor, in a separate PR.